docs/source/installing_dallinger_for_users.rst

```diff
@@ -91,3 +91,31 @@ you can move on.
 
 Next, you'll need :doc:`access keys for AWS, Heroku,
 etc. <aws_etc_keys>`.
+
+
+Install Heroku
+^^^^^^^^^^^^^^
+
+To run experiments locally or on the internet you need Heroku installed.
+You need at least version 3.28.0 to use Dallinger. You only need
+a Heroku account to launch experiments on the internet, you do not need it
+for local debugging.
+
+You can check your currently installed version by running:
+
+::
+
+    heroku --version
+
+Heroku is available for download from 
```

Comment:
@MatthewWilkes Ah, the spellchecker again. If you add the word to `docs/source/spelling_wordlist.txt` it'll get whitelisted. Might remove this — it's false alarming much more often than it is finding hits.